libnss-aad/docker/debian/Dockerfile

```
@@ -0,0 +1,39 @@
+FROM debian:9.7
```

Let's remove the `docker` subdirectory, e.g., `libnss_aad/debian/Dockerfile`

Also, let's bump this to Debian 11.0:

```suggestion
FROM debian:11.0
```